### PR TITLE
Boto/S3 tell support

### PR DESCRIPTION
**Pull request by Corni**
Provides a tell() implementation for boto/s3

### CODE REVIEW

File: simplekv/net/botostore.py [hunk 1]
```diff
@@ -87,10 +88,41 @@ def _get_filename(self, key, filename):
...
+
+        class SimpleKeyFile(KeyFile):
+            def read(self, size=-1):
+                if self.closed:
+                    raise ValueError("I/O operation on closed file")
+                if size < 0:
+                    size = self.key.size - self.location
+                return KeyFile.read(self, size)
+
+            def seekable(self):
```
Comment (fmarczin) [inline simplekv/net/botostore.py]: What is the purpose of these functions?

File: simplekv/net/botostore.py [hunk 2]
```diff
@@ -87,10 +88,41 @@ def _get_filename(self, key, filename):
...
+                return True
+
+            def readable(self):
+                return True
+
+            def seek(self, offset, whence=os.SEEK_SET):
+                if self.closed:
+                    raise ValueError("I/O operation on closed file")
+                if whence == 0:
```
Comment (fmarczin) [inline simplekv/net/botostore.py]: `0` (and `1` in L.114, `2` in L.118) should be the constants from `os`, like `os.SEEK_SET` above.

File: simplekv/net/botostore.py (hunk 1 above)
Comment (Corni) [reply simplekv/net/botostore.py]: See https://docs.python.org/3/library/io.html#io.IOBase.seekable

Comment (fmarczin) [reply simplekv/net/botostore.py]: 👍

**Comment by fmarczin** [comment]
👍 Thanks for the changes!

**Pull request merged**